Makefile

```diff
@@ -62,16 +62,17 @@ clean:
 # command. 'test' runs short tests that should last no more than a few seconds,
 # 'test-long' runs more thorough tests which should not last more than a few
 # minutes.
-pkgs = ./api ./compatibility ./crypto ./encoding ./modules ./modules/consensus \
-       ./modules/explorer ./modules/gateway ./modules/host ./modules/hostdb	   \
-       ./modules/miner ./modules/renter ./modules/transactionpool              \
-       ./modules/wallet  ./persist ./siac ./siae ./types
-test: clean fmt REBUILD
-	go test -short -tags='debug testing' -timeout=10s $(pkgs)
-test-v: clean fmt REBUILD
-	go test -race -v -short -tags='debug testing' -timeout=35s $(pkgs)
+run = Test
+pkgs = ./api ./build ./compatibility ./crypto ./encoding ./modules ./modules/consensus \
+       ./modules/explorer ./modules/gateway ./modules/host ./modules/hostdb \
+       ./modules/miner ./modules/renter ./modules/transactionpool \
+       ./modules/wallet ./persist ./siac ./siae ./types
+test: REBUILD
+	go test -short -tags='testing' -timeout=3s $(pkgs) -run $(run)
+test-v: REBUILD
+	go test -race -v -short -tags='debug testing' -timeout=15s $(pkgs) -run $(run)
 test-long: clean fmt REBUILD
-	go test -v -race -tags='testing debug' -timeout=300s $(pkgs)
+	go test -v -race -tags='testing debug' -timeout=300s $(pkgs) -run $(run)
```

Comment:
yeah but 80% of the time you don't use the extra stuff, vs with `go test` you'd end up writing everything out every time.